src/internal/Transition.js

```diff
@@ -1,7 +1,5 @@
 // @flow weak
-
```

Comment:
Point on which we need to agree. React [seems to be using](https://github.com/facebook/react/blob/master/src/renderers/shared/fiber/ReactDebugFiberPerf.js) a blank line. I think that this is increasing entropy regarding the codebase. What should we do?